tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# Runs the tests and creates a code coverage report.
+
+pytest --cov=virtual_ship --cov-report=html tests
```

Comment:
Also here add extra verbose/debug output
```suggestion
pytest -v --cov=virtual_ship --cov-report=html tests
```